<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medical Claims</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Images/Medical.png</normaloff>:/Images/Medical.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="PageExpenses" name="tabExpenses">
       <attribute name="title">
        <string>Expenses</string>
       </attribute>
      </widget>
      <widget class="PagePatients" name="tabPatients">
       <attribute name="title">
        <string>Patients</string>
       </attribute>
      </widget>
      <widget class="PageProviders" name="tabProviders">
       <attribute name="title">
        <string>Providers</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="actionOptions"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionDel"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionDuplicate"/>
   <addaction name="actionAutoFill"/>
   <addaction name="actionFilter"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Add.png</normaloff>:/Images/Add.png</iconset>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>Add record</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Exit.png</normaloff>:/Images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Minus.png</normaloff>:/Images/Minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete record</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Options.png</normaloff>:/Images/Options.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
   <property name="toolTip">
    <string>Options</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Save.png</normaloff>:/Images/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Export.png</normaloff>:/Images/Export.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export selected records</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Copy.png</normaloff>:/Images/Copy.png</iconset>
   </property>
   <property name="text">
    <string>Duplicate</string>
   </property>
   <property name="toolTip">
    <string>Duplicate selected rows</string>
   </property>
  </action>
  <action name="actionAutoFill">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/AutoFill.png</normaloff>:/Images/AutoFill.png</iconset>
   </property>
   <property name="text">
    <string>Auto fill</string>
   </property>
   <property name="toolTip">
    <string>Auto fill the selected cells</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Images/Filter.png</normaloff>:/Images/Filter.png</iconset>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
   <property name="toolTip">
    <string>Filter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PagePatients</class>
   <extends>QWidget</extends>
   <header>PagePatients.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageProviders</class>
   <extends>QWidget</extends>
   <header>PageProviders.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PageExpenses</class>
   <extends>QWidget</extends>
   <header>PageExpenses.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
